php/Blockonomics.php

```diff
@@ -702,6 +716,24 @@ public function get_order_info($order_id, $crypto){
         exit(json_encode($order));
     }
 
+    // Get the order info by id and crypto
+    public function get_order_amount_info($order_id, $crypto){
+        $order = $this->process_order($order_id, $crypto);
+
+        $order_amount = $this->fix_displaying_small_values($order['satoshi']);
+        
+        $cryptos = $this->getActiveCurrencies();
+        $crypto_obj = $cryptos[$crypto];
+
+        $response = array(
+            "payment_uri" => $this->get_crypto_payment_uri($crypto_obj, $order['address'], $order_amount),
```

Comment:
My first reaction is both order_amount and payment_uri can calculated from JS side easily avoiding us to pass extra parameters from server.

It looks like this was done for compatibility with noJS template ? This is fine. We just need to add code comment so that we can understand the motivation of the redundant params being passed from server